### DISCUSSION TITLE
How to uninstall nuclei?

### DISCUSSION THREAD

**Discussion by hassankhan98** [projectdiscovery/nuclei#593] [Q&A] [answered]
How to uninstall nuclei in macOS?

**Comment by Mzack9999** [comment, marked as answer]
If you installed it via brew:

```
brew uninstall nuclei
```

Otherwise it should be enough to remove the `nuclei` binary

**Comment by ehsandeep** [comment]
@hassankhan98 if you installed using `go get` you can also find the location of binary using `which nuclei` and remove the binary.
